driver.js

```diff
@@ -31,7 +31,7 @@ alice.showAllBalances();
 
 // The miners will start mining blocks when start is called.  After 5 seconds,
 // the code will terminate and show the final balances from Alice's perspective.
-bc.start(5000, () => {
+bc.start(8000, () => {
```

Comment:
Either 5 seconds or 8 is fine with me, but the comment and code should match.  Should we go with 5 seconds?

Reply:
I observed that if the delay is anything <8, the final balances from Alice's perspective do not contain Donald as there I assume the code finishes before Donald can be added to Alice's client list.